torch/utils/_config_module.py

```diff
@@ -25,7 +25,8 @@ def install_config_module(module: ModuleType) -> None:
     """
 
     class ConfigModuleInstance(ConfigModule):
-        _bypass_keys = set({"_is_dirty", "_hash_digest"})
+        # __annotations__ is written to by Sphinx autodoc
+        _bypass_keys = set({"_is_dirty", "_hash_digest", "__annotations__"})
```

Comment:
It turns out Sphinx writes to `__annotations__` when doing autodoc, but config module doesn't allow writes to that property by default. Now it does.